Jasmine/Common/Models/GameInfo.swift

```diff
@@ -0,0 +1,6 @@
+struct GameInfo {
```

Comment:
This is like a comparable object to GameData, merge them? or why should it be distinct